begin/cmdline.py

```diff
@@ -58,10 +58,16 @@ def from_name(self, name, default=NODEFAULT, section=None):
         "Get default value from argument name"
         if len(self._parser.sections()) > 0:
             section = self._section if section is None else section
-            try:
-                default = self._parser.get(section, name)
-            except (configparser.NoSectionError, configparser.NoOptionError):
-                pass
+            if isinstance(section, list):
+                sections = section
+            else:
+                sections = []
+                sections.append(section)
+            for section in sections:
```

Comment:
Ok.

Would be the case to assume that self._section is always a list and move all the type-checking logic to set_config_section()?

Also set_config_section() could be refactored to be a property setter for self.section. WDYT?